### ISSUE TITLE
hinawa-apogee-duet-cli errors with Debian testing

### ISSUE THREAD

**Issue by VennStone** [takaswie/hinawa-utils#5]
I upgraded one of my testing boxes to Debian Testing and noticed the hinawa-apogee-duet-cli no longer works. The Apogee Duet functions as normal with Jack2. 

Traceback (most recent call last):
  File "/usr/bin/hinawa-apogee-duet-cli", line 6, in <module>
    from hinawa_utils.oxfw.apogee_duet_unit import ApogeeDuetUnit
  File "/usr/lib/python3/dist-packages/hinawa_utils/oxfw/apogee_duet_unit.py", line 6, in <module>
    from hinawa_utils.oxfw.apogee_protocol import (
  File "/usr/lib/python3/dist-packages/hinawa_utils/oxfw/apogee_protocol.py", line 141, in <module>
    class InputCmd():
  File "/usr/lib/python3/dist-packages/hinawa_utils/oxfw/apogee_protocol.py", line 274, in InputCmd
    def get_meters(cls, unit: Hinawa.FwUnit):
  File "/usr/lib/python3/dist-packages/gi/module.py", line 123, in __getattr__
    raise AttributeError("%r object has no attribute %r" % (
AttributeError: 'gi.repository.Hinawa' object has no attribute 'FwUnit'

Thought I would pass it along.

**Comment by takaswie** [comment]
Hi,

> I upgraded one of my testing boxes to Debian Testing and noticed the hinawa-apogee-duet-cli no longer works.

I'm sorry for your inconvenience...

At first, Python 3 scripts in hinawa-utils are currently abandoned since the most of my time is used for [usespace control services for audio and music units on IEEE 1394 bus](https://github.com/alsa-project/snd-firewire-ctl-services/). Your Apogee Duet FireWire is continuously supported by one of the service programs.

Well, hinawa-utils depends on [libhinawa](https://github.com/alsa-project/libhinawa). I have released libhinawa 2.0 this year with [a slight loss of backward compatibility](https://github.com/alsa-project/libhinawa/releases/tag/2.0.0). The below message comes from the loss:

```
AttributeError: 'gi.repository.Hinawa' object has no attribute 'FwUnit'
```

As a quick glance, in Debian testing, [libhinawa 2.0.0 is installed](https://packages.debian.org/source/bullseye/libhinawa). On the other hand, [python3-hinawa-utils depends on gir1.2-hinawa-2.0](https://packages.debian.org/buster/python3-hinawa-utils) from libhinawa 1.0. This is expected.

I guess that you use hinawa-utils downloaded by hand. In the case, dependencies are not maintained by apt/dpkg and encounter the issue.

**Comment by VennStone** [comment]
I switched to unstable to see if the issue persists. hinawa-utils was installed directly from the the Debian repo. This is what is installed on the system with apt install hinawa-utils.

```
gir1.2-hinawa-3.0/unstable,unstable,now 2.0.0-1+b1 amd64 [installed,automatic]
  GObjet introspection data for libhinawa2

hinawa-utils/unstable,unstable,now 0.2.0-3 all [installed]
  Utilities to control Audio and Music units on FireWire (IEEE1394)

libhinawa-dev/unstable,unstable 2.0.0-1+b1 amd64
  I/O library for IEEE 1394 asynchronous transactions (development file)

libhinawa2/unstable,unstable,now 2.0.0-1+b1 amd64 [installed,automatic]
  I/O library for IEEE 1394 asynchronous transactions

python3-hinawa-utils/unstable,unstable,now 0.2.0-3 all [installed,automatic]
  Library to control Audio and Music units on FireWire (IEEE1394)
```

I did not know hinawa-utils was abandoned. Thank you for letting me know.
